### DISCUSSION TITLE
Running MESA with pypy

### DISCUSSION THREAD

**Discussion by subhamonsey** [mesa/mesa#1725] [General]
I am trying to speed up mesa and have been trying different ways. I tried using pypy but there seems to be a problem with pypy installing Pandas, even though their official website claim to be pandas compatible as of today. I looked at your PR threads and I think currently MESA doesn't run on pypy. Is that correct or am I missing something?

**Comment by rht** [comment]
Running on PyPy used to be tested in CI until it was too slow to install the dependencies: #1253 (April 2022). You can fork the Mesa main branch and comment out the pandas-specific code, if you still want to try it.

A more viable and less intrusive way to speed up Mesa would be to rewrite some of its components in Cython: https://github.com/projectmesa/mesa/discussions/1610. This has been tested, and just needs to be done.
